ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-release-4.10__nfd.yaml

```
@@ -0,0 +1,46 @@
+base_images:
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: build/Dockerfile
+  from: os
+  to: ci-artifacts
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 2Gi
+    requests:
+      cpu: 2000m
+      memory: 2Gi
+tests:
+- as: operator-e2e-master
+  cron: 0 23 * * *
+  steps:
+    cluster_profile: aws
+    test:
+    - as: nightly
+      commands: run prow nfd-operator test_master_branch
```

We just merged https://github.com/openshift-psap/ci-artifacts/pull/310
so this should be just `test_branch` 
```suggestion
      commands: run prow nfd-operator test_branch
```

---

Thank you, will adjust.